### PR TITLE
Renamed wrong namespace. Removed unused using directives #93

### LINKED ISSUE #93: Rename TinyHelpers.AspNetCore.Attributes namespace
The `TinyHelpers.AspNetCore.Attributes` namespace should be renamed to `TinyHelpers.AspNetCore.DataAnnotations`, to be aligned with the standard.

### PR DESCRIPTION
**Pull request by pampua84**
fixed #93

### CODE REVIEW

**Review by marcominerva — CHANGES_REQUESTED** [review]

File: samples/TinyHelpers.Sample/Program.cs [hunk 1]
```diff
@@ -15,4 +16,7 @@
 
 Console.ReadLine();
 
-public record class Person(string FirstName, string LastName, string City);
\ No newline at end of file
+namespace TinyHelpers.Sample
```
[the remark this answers is not in the document]
Comment (pampua84) [reply samples/TinyHelpers.Sample/Program.cs:19]: It was automatic refactoring at SonaLlint suggestion.

image

[the remark this answers is not in the document]
Comment (marcominerva) [reply samples/TinyHelpers.Sample/Program.cs:19]: Appliyng Sonarlint suggestions isn't in the scope of the issue, so I ask you ti revert this change.

**Review by marcominerva — CHANGES_REQUESTED** [review]
Please revert Sonarlint suggestion about Program.cs

**Review by pampua84 — COMMENTED** [review]
Reverted Sonarlint suggestion about Program.cs

**Review by pampua84 — COMMENTED** [review]
Reverted Sonarlint suggestion about Program.cs

**Comment by pampua84** [comment]
In this commit b9cd12c78caf81a5421bb3f53fb7da2790ac60a8 should i have solved the problem? or am I wrong?

**Comment by marcominerva** [comment]
Please mark the conversation as resolved and then request a new review.

**Pull request closed without merging**